CHANGES.md

```diff
@@ -69,6 +69,9 @@ Unreleased
 - Fix inline tests with *js_of_ocaml* and whole program compilation mode
   enabled (#6645, @hhugo)
 
+- Make bootstrap-info consistent for the threads library between 4.x and 5.x
```

Comment:
Since this isn't user facing, I don't think it warrants an entry here.

Reply:
Removed